exe/node/Main.hs

```diff
@@ -63,10 +64,9 @@ main = do
                  (progDesc "Oscoin Node")
 
     keys <-
-        flip runReaderT (Just $ keysDir optPaths) $
-            catches readKeyPair
+            catches (Crypto.readKeyPair $ keysDir optPaths </> "id")
                 [ Handler $ \case
-                    KeyNotFound{}
+                    Crypto.KeyNotFound{}
```

Comment:
At this point I'm wondering if we should just create a `Oscoin.Crypto` module which re-exports everything below 🤔

Reply:
I’d be all up for that.